playwright-e2e/functional/contested/create_case/create_case.spec.ts

```diff
@@ -43,7 +43,7 @@ test(
 
     // Sign in
     await manageCaseDashboardPage.visit()
-    await loginPage.login(config.applicant_solicitor.email, config.applicant_solicitor.password, config.manageCaseBaseURL);
+    await loginPage.loginWaitForPath(config.applicant_solicitor.email, config.applicant_solicitor.password, config.manageCaseBaseURL,config.loginPaths.cases);
```

Comment:
space after comma

Reply:
Thanks. Found and updated everywhere that I did this.